test/fixtures/es-module-specifiers/package-with-exports/exports_main.js

```diff
@@ -0,0 +1,3 @@
+const identifier = 'package-with-exports';
+console.log(identifier);
```

Comment:
remove `console.log` better, same to the following

Reply:
Sorry, it was copied from `test/fixtures/es-modules/package-type-module/index.js`, didn't notice it. I will fix it.